packages/react-vapor/src/components/editor/EditorConstants.ts

```diff
@@ -10,3 +10,5 @@ export const CodeMirrorModes = {
     Python: 'python',
     Text: 'text/plain',
 };
+
+export const DEFAULT_JSON_ERROR_MESSAGE: string = 'The JSON configuration is syntactically invalid.';

```

Comment:
is the `string` necessary? I thought it was implied and immutable since it's a const?

Reply:
Nice catch! I simply moved it from another file :see_no_evil: